cfpb/consumerfinance.gov
# PfC: Add `m-btn-group` for group of buttons | turn button class into button ID.

Wraps two button next to the each other in a `m-btn-group` (see https://cfpb.github.io/design-system/components/buttons#destructive-action). 

## Changes

- Wraps two button next to the each other in a `m-btn-group`
- Changes `btn__get-started` class to `college-costs-btn-get-started` ID.
- Changes `btn__next-step` class to `college-costs-btn-next-step` ID.


## How to test this PR

1. `yarn build` and visit http://localhost:8000/paying-for-college/your-financial-path-to-graduation/ and see that the navigation buttons still work and there is a greater gap between the buttons.


## Screenshots

Before:

<img width="289" alt="Screenshot 2024-05-02 at 10 27 18 AM" src="https://github.com/cfpb/consumerfinance.gov/assets/704760/44ef280f-8d96-4c10-86c5-6e0878ec01c1">


After:

<img width="290" alt="Screenshot 2024-05-02 at 10 27 49 AM" src="https://github.com/cfpb/consumerfinance.gov/assets/704760/59335deb-0349-4230-8179-266538ed02f0">
